src/notification-manager.coffee

```diff
@@ -46,8 +56,18 @@ class NotificationManager
   #
   # * `message` A {String} message
   # * `options` (optional) An options {Object} with the following keys:
-  #    * `detail` (optional) A {String} with additional details about the
-  #      notification.
+  #    * `buttons` (optional) An {Array} of {Object} where each {Object} has the
+  #      following options:
+  #      * `className` (optional) {String} additional class name add to the
+  #        button. It will already receive `btn btn-info`.
+  #      * `onDidClick` (optional) {Function} callback to call when the button
+  #        has been clicked. The context will be set to the
+  #        {NotificationElement} instance.
+  #      * `text` {String} inner text for the button
+  #    * `description` (optional) A {String} that will be rendered as Markdown
+  #      to describe the notification.
+  #    * `detail` (optional) A preformatted {String} that will be rendered as
```

Comment:
Do you think we could remove `preformatted` here? When reading it it felt like the string the user needs to pass has to be preformatted (i.e. be an HTML string or something). What about rewriting it as:

```
A plain-text {String} containing additional details about the notification.
```

Similarly we could update `description` like this:

```
A Markdown {String} with a longer description of the notification.
```

Thoughts? 💭